app/services/data/get-team-caseload.js

```diff
@@ -0,0 +1,6 @@
+const config = require('../../../knexfile').web
```

Comment:
Presuming you just created this file to get your tests to pass? 

Can you remove? - in case DJ uses a different file name and this one gets forgotten about and the tests pass when they shouldn't